test/test_gdalutilities.jl

```diff
@@ -18,93 +188,64 @@ AG.read("data/utmsmall.tif") do ds_small
-            @test AG.width(ds_small) == 100
-            @test AG.height(ds_small) == 100
-            @test AG.shortname(AG.getdriver(ds_small)) == "GTiff"
-            @test AG.width(ds_warped) == 100
-            @test AG.height(ds_warped) == 100
-            @test AG.shortname(AG.getdriver(ds_warped)) == "MEM"
-        end
+    @testset "GDAL Build VRT" begin
+        ds_vrt = AG.unsafe_gdalbuildvrt([ds_tiny])
+        @test AG.read(ds_vrt, 1) == [128  171  127   93   83;
+                                        126  164  148  114  101;
+                                        161  175  177  164  140;
+                                        185  206  205  172  128;
+                                        193  205  209  181  122]
     end
-end
 
-AG.read("data/point.geojson") do ds_point
-    @testset "GDAL Grid" begin
-        AG.gdalgrid(ds_point, ["-of","MEM","-outsize","3",
-            "10","-txe","100","100.3","-tye","0","0.1"]) do ds_grid
-            @test AG.getgeotransform(ds_grid) ≈ [100.0,0.1,0.0,0.0,0.0,0.01]
-        end
+    @testset "GDAL DEM Processing" begin
+        ds_dempr = AG.unsafe_gdaldem(ds_tiny, "hillshade", ["-of","AAIGrid"])
+        @test AG.read(ds_dempr, 1) == [ 0    0    0    0  0;
+                                        0  183  180  181  0;
+                                        0  184  182  181  0;
+                                        0  183  181  177  0;
+                                        0    0    0    0  0]
     end
 
-    @testset "GDAL Rasterize" begin
-        AG.gdalrasterize(ds_point, ["-of","MEM","-tr","0.05","0.05"]) do ds_rasterize
-            @test AG.getgeotransform(ds_rasterize) ≈ [99.975,0.05,0.0,0.1143,0.0,-0.05]
-        end
+    @testset "GDAL Near Black" begin
+        ds_nearblack = AG.unsafe_gdalnearblack(ds_tiny, ["-of","GTiff","-color","0"])
+        @test AG.read(ds_nearblack, 1) == [ 0  0    0  0  0;
+                                            0  0    0  0  0;
+                                            0  0  177  0  0;
+                                            0  0    0  0  0;
+                                            0  0    0  0  0]
     end
 
-    @testset "GDAL Vector Translate" begin
-        AG.gdalvectortranslate([ds_point], ["-f","CSV","-lco",
-            "GEOMETRY=AS_XY"], dest = "data/point.csv") do ds_csv
+    # cannot reproject file on AppVeyor yet
+    # GDALError (CE_Failure, code 4):
+    #       Unable to open EPSG support file gcs.csv.  Try setting the
+    #       GDAL_DATA environment variable to point to the directory
+    #       containing EPSG csv files.
+    # @testset "GDAL Warp" begin
+    #     AG.gdalwarp([ds_small], ["-of","MEM","-t_srs","EPSG:4326"]) do ds_warped
+    #         @test AG.width(ds_small) == 100
+    #         @test AG.height(ds_small) == 100
+    #         @test AG.width(ds_warped) == 109
+    #         @test AG.height(ds_warped) == 91
+    #     end
+    # end
```

Comment:
I assume this test can be enabled now?

Reply:
Done in https://github.com/yeesian/ArchGDAL.jl/pull/179/commits/ac50e17d930a19142b9d0b9ba79e2e5b13a936ce, thanks for the catch!